src/statistics.js

```diff
@@ -0,0 +1,118 @@
+import { Stats } from "./Stats.js";
+import { categories, state } from "./index.js";
+import { toStatsView } from "./navigation.js";
+import { sortTable } from "./sort_statistics.js";
+
+function createStats() {
+  const tbody = document.querySelector("#stats-table tbody");
+
+  for (const category of categories) {
+    for (const card of category.cards) {
+      const row = createRow(card);
+      tbody.append(row);
+    }
+  }
+  addResetListener();
+  const tableHeaders = tbody.querySelectorAll("th");
+  for (const [i, header] of tableHeaders.entries()) {
+    header.addEventListener("click", () => sortTable(i));
+  }
+}
+
+function getStats(key) {
+  const stats = JSON.parse(localStorage.getItem(key)) || new Stats(key);
+  return stats;
+}
```

Comment:
The function can be simplified because we use a variable only to return it (we call it an "inline variable"). It doesn't improver the readability of the code and just adds redundant lines.

```suggestion
function getStats(key) {
  return JSON.parse(localStorage.getItem(key)) || new Stats(key);
}
```